### PR TITLE
PMM-12165 Changed queries for new plugin

### PR DESCRIPTION
**Pull request by anton-bystrov**
PMM-12165 

https://github.com/Percona-Lab/pmm-submodules/pull/3293

This PR:
1/ solves the problem of broken queries, which led to the absence of metrics on Environment Dashboard
2/ replaces boomtable with the native table plugin

This is how the Environment Overview looks now:
Screenshot 2023-06-09 at 03 43 43

Screenshot 2023-06-09 at 03 44 06

### CODE REVIEW

**Review by 0leksii — DISMISSED** [review]

File: dashboards/Experimental/Environments_Overview.json [hunk 1]
```diff
@@ -88,16 +89,36 @@
...
+        {
+          "datasource": {
+            "0": "M",
+            "1": "e",
+            "2": "t",
+            "3": "r",
+            "4": "i",
+            "5": "c",
+            "6": "s"
+          },
```
Comment (atymchuk) [inline dashboards/Experimental/Environments_Overview.json:112]: This looks a bit strange. Why is it broken up by letters?

File: dashboards/Experimental/Environments_Overview.json [hunk 2]
```diff
@@ -948,7 +1036,19 @@
       "steppedLine": true,
       "targets": [
         {
-          "datasource": "Metrics",
+          "datasource": {
```
Comment (atymchuk) [inline dashboards/Experimental/Environments_Overview.json:1039]: Can this be one string, e.g. `"datasource": "-- Mixed --"`?

File: dashboards/Experimental/Environments_Overview.json [hunk 3]
```diff
@@ -1385,24 +1530,158 @@
...
+            "table": "metrics"
+          },
+          "datasource": {
+            "type": "grafana-clickhouse-datasource",
+            "uid": "PDEE91DDB90597936"
+          },
+          "format": 0,
+          "hide": false,
+          "meta": {
+            "builderOptions": {
```
Comment (atymchuk) [inline dashboards/Experimental/Environments_Overview.json:1610]: Can this be just one string? It doesn't feel right either.

File: dashboards/Experimental/Environments_Overview.json [hunk 4]
```diff
@@ -1421,6 +1700,7 @@
       },
       "yaxes": [
         {
+          "$$hashKey": "object:73",
```
Comment (atymchuk) [inline dashboards/Experimental/Environments_Overview.json:1703]: We need to remove those $$hasKey entries - they are redundant and not good for maintenance.

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Review by atymchuk — DISMISSED** [review]

**Review by BupycHuk — DISMISSED** [review]

**Pull request merged**